server/routes/locations.js

```diff
@@ -0,0 +1,44 @@
+const { isNilOrEmpty } = require('../utils/utils')
+const { processInput } = require('../services/validation')
+const { full } = require('../config/incident')
+
+module.exports = ({ locationService }) => {
+  // get list of all prisons
```

Comment:
No point having comments if they don't add any context as to why a decision was made or explaining complicated or surprising code that you can't express intent by rewriting/simplifying the code. 

Generally a well named method can replaces the need for a comment. 

In this case renaming the route along the line of use case `changePrison` means you don't really need the comment at all. If that code is executed then you are `"submit"ing a change of prison`

Reply:
ok